resources/qml/Dialogs/DiscardOrKeepProfileChangesDialog.qml

```diff
@@ -146,12 +165,28 @@ UM.Dialog
             id: discardButton
             text: catalog.i18nc("@action:button", "Discard changes")
             onClicked: base.accept()
+            visible: alternateStates.state == ""
         },
         Cura.SecondaryButton
         {
             id: keepButton
             text: catalog.i18nc("@action:button", "Keep changes")
             onClicked: base.reject()
+            visible: alternateStates.state == ""
+        },
+        Cura.SecondaryButton
+        {
+            id: overwriteButton
+            text: catalog.i18nc("@action:button", "Save as new custom profile")
+            onClicked: base.accept()
+            visible: alternateStates.state != ""
```

Comment:
```suggestion
            visible: alternateStates.state != ""
            onClicked: base.accept()
```

It's not in the style guide yet, but I think I will add it now. 

It's easier to read if you always order the properties id, width/height/layout, style, functions